test-data/unit/pythoneval-asyncio.test

```diff
@@ -269,15 +269,16 @@ def greet() -> 'Generator[Any, None, None]':
 @asyncio.coroutine
 def test() -> 'Generator[Any, None, None]':
     yield from greet()
-    x = yield from greet()  # Error
+    x = yield from greet()
```

Comment:
For consistency, we should retain the original error message (... does not return a value).